frontend/src/app/components/ui-components/record-edit-fields/text/text.component.html

```
@@ -8,8 +8,8 @@
         [validateType]="validateType"
         [regexPattern]="regexPattern"
         attr.data-testid="record-{{label()}}-text"
-        [(ngModel)]="value" (ngModelChange)="onFieldChange.emit($event)">
-    @if (maxLength && maxLength > 0 && value() && (maxLength - value().length) < 100) {
+        [(ngModel)]="value" (ngModelChange)="handleValueChange($event)">
+    @if (maxLength && maxLength > 0 && value() && 100 > (maxLength - value().length)) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)